sse2neon.h

```diff
@@ -4893,6 +4893,21 @@ FORCE_INLINE __m128i _mm_cvtps_epi32(__m128 a)
 #endif
 }
 
+// Converts the four single-precision, floating-point values of a to signed
```

Comment:
Take the description from https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtps_pi16
That is,
> Convert packed single-precision (32-bit) floating-point elements in a to packed 16-bit integers, and store the results in dst. Note: this intrinsic will generate 0x7FFF, rather than 0x8000, for input values between 0x7FFF and 0x7FFFFFFF.